cp-quickstart/start-docker-cloud.sh

```diff
@@ -5,95 +5,169 @@
 # To avoid unexpected charges, carefully evaluate the cost of resources before launching the script and ensure all resources are destroyed after you are done running it.
 #########################################
 
+NAME=`basename "$0"`
+LOG_FILE="$NAME.log"
+
+printf "\n>>>>>>>>>> starting %s @ %s >>>>>>>>>>\n\n" $NAME "$(date)" >> "$LOG_FILE"
```

Comment:
@rspurgeon I'm on the fence about this PR introducing logging.  Things to think about:

1. Is it necessary (enough)?
2. If necessary, should it be done in this PR or would it be better to make a swoop through all demos in a concerted effort?
3. Consequences of having a one-off demo with the logging behavior, whereas the rest do not?

What would you think about saving this for a separate PR?

Reply:
As I'm running the demo now: there's a lot more discussion we should have on what should or shouldn't be logged, which I'd prefer to not make the focus on this PR.  It should be a separate PR and maybe even benefit from a wiki 1-pager.  So I think we should remove logging from this PR.